src/TwoFactorAuthenticationPlugin.php

```diff
@@ -47,21 +50,44 @@ public function register(Panel $panel): void
                 ]);
         }
 
+        $panel
+            ->authMiddleware([
+                $this->challengeTwoFactorCheckMiddleware,
```

Comment:
Can we rename the `challengeTwoFactorCheckMiddleware` method to `twoFactorChallengeMiddleware`